packages/common/src/projects/defaults.ts

```diff
@@ -31,6 +32,7 @@ export const DEFAULT_PROJECT_MODEL: IModel = {
     description: "",
     snippet: "",
     tags: [],
+    categories: [],
```

Comment:
I'm not sure adding these defaults is actually necessary. It looks like when an item is created, it already defaults categories to an empty array, so this might be redundant.